docs/Makefile

```diff
@@ -1,10 +1,18 @@
 DOCS_FOLDER=$(shell git rev-parse --show-toplevel)/docs
 IMAGE_REGISTRY ?= public.ecr.aws/eks-distro-build-tooling
 CONTAINER_IMAGE   = $(IMAGE_REGISTRY)/eks-a-hugo
-CONTAINER_RUN     = docker run --rm --interactive --tty --volume $(CURDIR)/../:/src
 DOCS_ARCHIVE      = public.zip
 HTMLTEST_VERSION?=v0.16.0
 
+# Detach can be used with the container-serve command to 
```

Comment:
to what?

Reply:
Somethin like this?:  ...detach the container from the current shell and run it in the background.